templates/default/expat-site.xml.erb

```diff
@@ -61,13 +61,27 @@
         <pass><%= node['elastic']['opendistro_security']['admin']['password'] %></pass>
         <services-logs-user><%= node['elastic']['opendistro_security']['service_log_viewer']['username'] %></services-logs-user>
         <services-logs-pass><%= node['elastic']['opendistro_security']['service_log_viewer']['password'] %></services-logs-pass>
+        <snapshot>
+            <repo>
+                <name></name>
+                <location></location>
+            </repo>
+            <name></name>
+            <!-- repeated section for multiple indices -->
+            <indices></indices>
+            <ignoreUnavailable></ignoreUnavailable>
+        </snapshot>
     </elastic>
 
     <hops>
         <client>
             <user><%= node['hops']['hdfs']['user'] %></user>
         </client>
     </hops>
+
+    <epipe>
+        <path></path>
```

Comment:
can you add the epipe base dir variable to the template

Reply:
fixed